XBVC/emitters/templates/c_header.jinja2

```diff
@@ -52,6 +52,9 @@ void xbvc_init(void *params,
 	       xbvc_platform_init_handler platform_init);
 
 void xbvc_run(void);
-int xbvc_send(void *msg, uint32_t msg_type);
-	
+int xbvc_send(void *msg, uint32_t msg_type, uint32_t *rnd_id_ret);
```

Comment:
Is there a reason the xbvc_send and xbvc_send_response can't generate the random number?